tests/realtikvtest/txntest/txn_state_test.go

```diff
@@ -119,7 +119,7 @@ func TestBasicTxnState(t *testing.T) {
 	require.Nil(t, info)
 }
 
-func TestEntriesCountAndSize(t *testing.T) {
+func TestEntriesCount(t *testing.T) {
```

Comment:
Why not check Size?

Reply:
Now `size` is checked by the new test because it now reflects the actual memory footprint and thus is not a fixed number.